### PR TITLE
Update CI name

### PR DESCRIPTION
**Pull request by kevmoo**
...and force a CI restart

### CODE REVIEW

File: .github/workflows/test-package.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Dart CI
+name: ci
```
Comment (devoncarew) [inline .github/workflows/test-package.yml:1]: Just FYI, this is the name that will show in the markdown badge.

**Review by devoncarew — APPROVED** [review]

File: .github/workflows/test-package.yml (hunk 1 above)
Comment (kevmoo) [reply .github/workflows/test-package.yml:1]: 🤷  – do you like the other name more?

Comment (kevmoo) [reply .github/workflows/test-package.yml]: just updated the action version instead 😄

Comment (devoncarew) [reply .github/workflows/test-package.yml]: `Dart`? No particular preference, just fyi, that the markdown badge's name would have gotten short :)

**Pull request merged**